dd-sdk-android/src/main/kotlin/com/datadog/android/core/configuration/Configuration.kt

```diff
@@ -151,6 +152,21 @@ internal constructor(
             )
         }
 
+        /**
+         * Sets the DataDog SDK to be more verbose when an application is set to `debuggable`.
+         * This is equivalent to setting:
+         *   sampleRumSessions(100)
+         *   setBatchSize(BatchSize.SMALL)
+         *   setUploadFrequency(UploadFrequency.FREQUENT)
+         *   Datadog.setVerbosity(Log.VERBOSE)
+         * These settings will override your configuration, but only when the application is `debuggable`
+         * @param verboseDebugInfo Enable or disable verbose info when debugging
+         */
+        fun setVerboseDebugInfo(verboseDebugInfo: Boolean): Builder {
```

Comment:
I think we should come up with another name. `setVerboseDebugInfo` only hints that it will do `Datadog.setVerbosity(Log.VERBOSE)`, but we're making more changes. We could make this a bit more generic with a name like `enableDeveloperModeWhenDebuggable` or something in the same sense.

Reply:
Yeah, that's a much better name.